File: Phasegate/DayOne/Level 1/java/TEST1/TaskOne.java
public class TaskOne{
  public static int [] seprateEvenandOdd(int [] numbers){
    int even = 0;
    int odd = 0;
    for(int count = 0; count < numbers.length; count++){
      if(numbers[count] % 2 == 0){
        even++;
      }
      else{
        odd++;
      }
    }
    int evenCount = new int[even];
    int oddCount = new int[odd];
    int eveCount = 0;
    int oddCount = 0;
    for(int index = 0; index < numbers.length; index++){
      if(numbers[index] % 2 == 0){
        even[evenCount] = numbers[index];
        evenCount++;
      }
      else{
        odd[oddCount] = numbers[index];
        oddCount++;
      }
    }
    reurn new int [][]{even,odd];
 }
}  

File: Phasegate/DayOne/Level 1/python/TEST/task_one.py
def even_numbers(numbers):
    even = []
    for num in numbers:
        if num % 2 == 0:
            even.append(num) 
    return even
def odd_numbers(numbers):  
    odd = []   
    for num in numbers:               
        if num % 2 != 0:
            odd.append(num) 
    return odd
